schematron/v2.2.0/exemplary_files/L200_Audit.xml

```diff
@@ -637,25 +644,28 @@
             </auc:Building>
           </auc:LinkedPremises>
         </auc:Schedule>
-        <auc:Schedule ID="ProcessLoadSchedule">
+        <auc:Schedule ID="HVACSchedule">
           <auc:ScheduleDetails>
             <auc:ScheduleDetail>
               <auc:DayType>Weekday</auc:DayType>
-              <auc:ScheduleCategory>Miscellaneous equipment</auc:ScheduleCategory>
+              <auc:ScheduleCategory>HVAC equipment</auc:ScheduleCategory>
               <auc:DayStartTime>09:00:00</auc:DayStartTime>
               <auc:DayEndTime>17:00:00</auc:DayEndTime>
+              <auc:PartialOperationPercentage>80</auc:PartialOperationPercentage>
```

Comment:
@nllong - for an occupancy schedule, we use the concept of partial operation percentage to indicate the % of design capacity we expect for this time.  I think this also makes sense for something like an equipment schedule, but what about for an HVAC schedule?  Does this have any mrit?

Reply:
No, for HVAC systems it should just be 'available' or 'not available'. However, for components of HVAC it could be partial operation.

Reply:
@corymosiman12 I'm assuming this means I don't need to change the requirements?

Reply:
So Nick, this could go one of two ways:
1. We create a new element as part of a schedule which is basically an 'on/off' OR 'available/unavailable' enumeration
2. We assert that for certain things the `auc:PartialOperationPercentage` is either 0% or 100% (i.e. on/off)

?

Reply:
I like option (1) -- basically schedule type. Double check what is in e+, but I'm thinking, fraction, value, on/off, boolean, positive integers, integer, etc etc etc.

Reply:
Hey @nllong, so from what I found, E+ splits these into two distinct concepts:

1. The `ScheduleTypeLimits` object, which appears always to be a numeric type (but either Continuous or Discrete).  It then has upper / lower limits
2. A `Schedule` object.  The Schedule object references a ScheduleTypeLimits object, which is used for validation of the values defined within a schedule.

From my understanding, all the schedule types you describe are derived from the `ScheduleTypeLimits`:
- A fractional schedule type would be a ScheduleTypeLimits that is **continuous** with lower bound = 0, upper bound = 1
- An on/off schedule type would be a ScheduleTypeLimits that is **discrete** with lower bound = 0, upper bound = 1

The way our `auc:ScheduleType` element in BSync is specified, it doesn't really provide for this mechanism.  Let's discuss in our next call on how to proceed.

Reply:
@macintoshpie @corymosiman12 come up with a proposal or two for doing better schedules.

Reply:
Moved to #260